vendor/bat-native-ledger/src/bat_helper.h

```diff
@@ -354,12 +370,22 @@ namespace braveledger_bat_helper {
     std::string ruleset_;
     std::string rulesetV2_;
     BatchVotes batch_;
-    GRANT grant_;
+    Grants grants_;
     CurrentReconciles current_reconciles_;
     bool auto_contribute_ = false;
     bool rewards_enabled_ = false;
   };
 
+  struct GRANTS_PROPERTIES_ST {
```

Comment:
why is a vector of grant responses called `GRANTS_PROPERTIES_ST`? Also we should stop using this all caps naming

Reply:
I was following the naming conventions of the other response structs, I can update